phpstan.neon.dist

```diff
@@ -3,6 +3,7 @@ parameters:
     phpVersion: 80200
     paths:
         - src
+        - static-analysis
```

Comment:
Oh wow woopsie

Reply:
I guess this particular change should target 3.5.x?

Reply:
> Oh wow woopsie

Yeah, the one test was only run with Psalm previously. I don't think that this is a big deal tbh.

> I guess this particular change should target 3.5.x?

I don't think that's necessary.

Reply:
Ah indeed, if it's running with Psalm, it's not a bug then, we can view this as an improvement.